pom.xml

```diff
@@ -50,80 +50,76 @@
 		<openMRSVersion>1.9.9</openMRSVersion>
         <uiframeworkVersion>3.4</uiframeworkVersion>
 		<uilibraryVersion>1.5</uilibraryVersion>
+		<webservicesRestVersion>2.5</webservicesRestVersion>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 	</properties>
 
-	<dependencyManagement>
```

Comment:
getting rid of the dependency management stuff and just adding as dependencies here